src/CitMovie.Data/JobCategoryRepository/JobCategoryRepository.cs

```diff
@@ -0,0 +1,28 @@
+using CitMovie.Models.Dto;
+using Microsoft.EntityFrameworkCore;
+
+namespace CitMovie.Data.JobCategoryRepository
+{
+    public class JobCategoryRepository : IJobCategoryRepository
+    {
+        private readonly DataContext _context;
+
+        public JobCategoryRepository(DataContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IEnumerable<JobCategory>> GetAllJobCategoriesAsync(int page, int pageSize)
+        {
+            return await _context.JobCategories
```

Comment:
Here kind of the same as last PR, we do not expect to update these only read them, so we can optimize this by letting EF know that with:

```suggestion
            return await _context.JobCategories
            .AsNoTracking()
```